Source/Core/Common/x64Emitter.cpp

```diff
@@ -1683,11 +1754,13 @@ void XEmitter::CMP_or_TEST(int bits, const OpArg& a1, const OpArg& a2)
   CheckFlags();
   if (a1.IsSimpleReg() && a2.IsZero())  // turn 'CMP reg, 0' into shorter 'TEST reg, reg'
   {
-    WriteNormalOp(bits, NormalOp::TEST, a1, a1);
+    EmitJccFusableInstruction(JccFusionType::All,
+                              [&] { WriteNormalOp(bits, NormalOp::TEST, a1, a2); });
```

Comment:
Copy-paste error, last parameter should be `a1`.

Reply:
Thanks.